app/models/host.rb

```diff
@@ -11,6 +11,8 @@ class Host < ApplicationRecord
   sortable_by :name, :display_name
   sortable_by :os_major_version, OS_MAJOR_VERSION
   sortable_by :os_minor_version, OS_MINOR_VERSION
+  sortable_by :ssg_version, Arel.sql('benchmarks.version'),
```

Comment:
Would this sort naturally?

Reply:
Good catch, it doesn't inherit from the benchmarks model (would be nice) ... fixed :wrench: